submissions/finalise.R

```diff
@@ -69,30 +74,44 @@ submission <- rbindlist(list(submission, cum_submission))
 
 # Checks ------------------------------------------------------------------
 
+print("Checked for NAs, 0s, and comparing forecast with population")
+
 # adjust to total population if greater than
 state_pop <- fread(here("data", "state_pop.csv"))
 state_pop <- state_pop[, .(location = state_code, pop = tot_pop)]
 submission <- merge(submission, state_pop, by = "location")
 submission <- submission[pop < value, `:=`(value = pop - 1, value_exceeds_pop = 1)]
 
 if (sum(submission$value_exceeds_pop, na.rm = TRUE) > 0) {
-  warning("Forecast values exceed total population")
-  print(submission[value_exceeds_pop == 1])
+  warning(paste("Forecast values exceed total population in locations:",
+          unique(submission[value_exceeds_pop == 1, location])), ". 
+          Forecast values set to population. Consider excluding location")
```

Comment:
we aren't excluding location anymore - we need to fix rather than exclude (if we just exclude we get dropped from scoring).